analyzers/SonarAnalyzer.sln

```diff
@@ -52,7 +50,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Scripts", "Scripts", "{C086
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "AnalysisConfig", "AnalysisConfig", "{E044950A-098B-4090-B4FB-8D6BD086828B}"
 	ProjectSection(SolutionItems) = preProject
-		.sonarlint\sonaranalyzer-dotnetCSharp.ruleset = .sonarlint\sonaranalyzer-dotnetCSharp.ruleset
```

Comment:
What is this?

Reply:
Back in the day, when SonarLint used intrusive connected mode, this file used to hold the list of rules activated in SC quality profile. It has been deleted a long time ago.